### ISSUE TITLE
Non descriptive error message when requesting test call

### ISSUE THREAD

**Issue by relleums** [fact-project/shifthelper_webinterface#16]
When no phone number is entered (empty string) in the logbook, the shift helper web page says just 'internal server error' when requesting a test call. 
On the other hand, the error message is very good and descriptive when the phone number format is bad.

**Comment by maxnoe** [comment]
Is this still an issue? I thought this was fixed by https://github.com/fact-project/shifthelper_webinterface/commit/dc2c498f85b0520e042ca70b2ee307d8d210f474 which is already deployed.

**Comment by relleums** [comment]
We just encountered this right now with Axel

**Comment by maxnoe** [comment]
If it was just now, it was a connection error. I don't think that has something to do with the phone number being empty:


```
shifthelper_webinterface_1  | [2016-12-06 17:21:52,963] ERROR in app: Exception on /testCall [GET]
shifthelper_webinterface_1  | Traceback (most recent call last):
shifthelper_webinterface_1  |   File "/opt/conda/lib/python3.5/site-packages/pymysql/connections.py", line 1004, in _write_bytes
shifthelper_webinterface_1  |     self._sock.sendall(data)
shifthelper_webinterface_1  | BrokenPipeError: [Errno 32] Broken pipe
shifthelper_webinterface_1  |
shifthelper_webinterface_1  | During handling of the above exception, another exception occurred:
shifthelper_webinterface_1  |
shifthelper_webinterface_1  | Traceback (most recent call last):
shifthelper_webinterface_1  |   File "/opt/conda/lib/python3.5/site-packages/sqlalchemy/engine/base.py", line 1182, in _execute_context
shifthelper_webinterface_1  |     context)
shifthelper_webinterface_1  |   File "/opt/conda/lib/python3.5/site-packages/sqlalchemy/engine/default.py", line 462, in do_execute
shifthelper_webinterface_1  |     cursor.execute(statement, parameters)
shifthelper_webinterface_1  |   File "/opt/conda/lib/python3.5/site-packages/pymysql/cursors.py", line 166, in execute
shifthelper_webinterface_1  |     result = self._query(query)
shifthelper_webinterface_1  |   File "/opt/conda/lib/python3.5/site-packages/pymysql/cursors.py", line 322, in _query
shifthelper_webinterface_1  |     conn.query(q)
shifthelper_webinterface_1  |   File "/opt/conda/lib/python3.5/site-packages/pymysql/connections.py", line 834, in query
shifthelper_webinterface_1  |     self._execute_command(COMMAND.COM_QUERY, sql)
shifthelper_webinterface_1  |   File "/opt/conda/lib/python3.5/site-packages/pymysql/connections.py", line 1054, in _execute_command
shifthelper_webinterface_1  |     self._write_bytes(packet)
shifthelper_webinterface_1  |   File "/opt/conda/lib/python3.5/site-packages/pymysql/connections.py", line 1006, in _write_bytes
shifthelper_webinterface_1  |     raise err.OperationalError(2006, "MySQL server has gone away (%r)" % (e,))
shifthelper_webinterface_1  | pymysql.err.OperationalError: (2006, "MySQL server has gone away (BrokenPipeError(32, 'Broken pipe'))")
```

**Comment by maxnoe** [comment]
should also be fixed in #24
